v-next/hardhat/src/internal/cli/init/init.ts

```diff
@@ -119,15 +119,15 @@ export async function initHardhat(options?: InitHardhatOptions): Promise<void> {
 
 // generated based on the "DOS Rebel" font
 function printAsciiLogo() {
-  const logoLines = `                   
-█████   █████                          ███   ███                   ███         ███████
-░░███   ░░███                          ░███  ░███                  ░███        ██░░░░███
-░███    ░███   ██████   ████████   ███████  ░███████    ██████   ███████     ░░░   ░███
-░███████████  ░░░░░███ ░░███░░███ ███░░███  ░███░░███  ░░░░░███ ░░░███░          ████░
-░███░░░░░███   ███████  ░███ ░░░ ░███ ░███  ░███ ░███   ███████   ░███          ░░░░███
-░███    ░███  ███░░███  ░███     ░███ ░███  ░███ ░███  ███░░███   ░███ ███   ██    ░███
-█████   █████░░████████ █████    ░░████████ ████ █████░░████████  ░░█████   ░░████████
-░░░░░   ░░░░░  ░░░░░░░░ ░░░░░      ░░░░░░░░ ░░░░ ░░░░░  ░░░░░░░░    ░░░░░     ░░░░░░░░
+  const logoLines = `
+ █████  █████                          ███   ███                   ███       ████████
+░░███  ░░███                          ░███  ░███                  ░███      ███░░░░███
+ ░███   ░███   ██████   ████████   ███████  ░███████    ██████   ███████   ░░░    ░███
+ ░██████████  ░░░░░███ ░░███░░███ ███░░███  ░███░░███  ░░░░░███ ░░░███░       ██████░
+ ░███░░░░███   ███████  ░███ ░░░ ░███ ░███  ░███ ░███   ███████   ░███       ░░░░░░███
+ ░███   ░███  ███░░███  ░███     ░███ ░███  ░███ ░███  ███░░███   ░███ ███  ███   ░███
+ █████  █████░░████████ █████    ░░████████ ████ █████░░████████  ░░█████  ░░████████
+░░░░░   ░░░░░  ░░░░░░░░ ░░░░░      ░░░░░░░░ ░░░░ ░░░░░  ░░░░░░░░    ░░░░░    ░░░░░░░░
```

Comment:
I restored the leading newlines here that must have got lost. I also made the `H` 1 cell more narrow which is more consistent with the rest of the letters. Finally, I copied `3` from the website again because it was wonky after the other changes. 

Now I see that you made the part of the `3` on the left-hand side 2 cell wide while on the right it was 3 cell wide. I think I lost it on copy. Let me know if I should restore it and sorry for messing with the design.